app.js

```diff
@@ -16,7 +16,7 @@ const jwt = require('json-web-token');
 
 const env = process.env.NODE_ENV || 'development';
 
-if (env === 'development') {
+if (env === 'development' || 'test') {
```

Comment:
This won't work since the second half of the conditional check will always evaluate to true (`'test'` is truthy).

We'll need to explicitly check against `env` for both pieces of the conditional check.

```suggestion
if (env === 'development' || env === 'test') {
```

Reply:
Good catch